src/types/inc/utils.hpp

```diff
@@ -52,6 +52,20 @@ namespace Microsoft::Console::Utils
     bool StringToUint(const std::wstring_view wstr, unsigned int& value);
     std::vector<std::wstring_view> SplitString(const std::wstring_view wstr, const wchar_t delimiter) noexcept;
 
+    enum PasteOption
+    {
+        // Convert Windows-space \r\n and \n line-endings to \r line-endings.

```

Comment:
```suggestion
        // Convert Windows \r\n and \n line-endings to \r line-endings.
```

that is really funny, actually, that it breaks VS. ha!

if you want, you can say...

```
Convert CR+LF and LF-only line endings to CR-only.
```